src/picomsg/src/system_enterprise_map/c_api/system_enterprise_map.h

```diff
@@ -9,6 +9,9 @@ extern "C" {
 #include <stdint.h>
 #include "common_c_api/common.h"
 
+// Neighbor sub interface with exactly that many characters is *allowed*
```

Comment:
If the size of the `neigh_sub_interface` buffer is exactly `MAX_NEIGH_SUB_INTERFACE_LEN` then it is not allowed to have an interface with exactly that many characters..

Reply:
Daniel, again, **do not mark a comment as resolved if you didn't fix the issue**.

Reply:
Sorry, git problems :(
I've changed the size of neigh_sub_interface to:
`	char neigh_sub_interface[MAX_NEIGH_SUB_INTERFACE_LEN + 1];`